crates/core/src/diff/diff.rs

```diff
@@ -1,12 +1,14 @@
-use super::{MoveTo, Patch};
-use crate::dom::*;
-use smallvec::{smallvec, SmallVec};
 use std::{
     cmp::Ordering,
     collections::{BTreeMap, BTreeSet},
     fmt, mem,
     ops::Deref,
 };
+
+use smallvec::{smallvec, SmallVec};
+
+use super::{MoveTo, Patch};
+use crate::dom::*;
```

Comment:
I would normally put a blank line between the last use and the first declaration. Can you add that manually even though it wasn't there before?

Reply:
Makes sense; I always do it that way too